be/src/olap/rowset/segment_v2/column_reader.cpp

```diff
@@ -240,7 +258,9 @@ Status ColumnReader::_init_column_zone_map() {
     if (_meta.has_zone_map_page()) {
         PagePointer pp = _meta.zone_map_page();
         PageHandle ph;
-        RETURN_IF_ERROR(read_page(pp, &ph));
+        // tmp statistics
+        OlapReaderStatistics stats;
```

Comment:
Same here. this `stats` seems just a temporary variable?

Reply:
yes.